src/libraries/System.Net.Primitives/tests/UnitTests/CookieContainerTest.cs

```diff
@@ -762,5 +763,66 @@ private void VerifyGetCookies(CookieContainer cc1, Uri uri, Cookie[] expected)
                 Assert.Equal(c1.Value, c2.Value);
             }
         }
+
+        [Theory]
+        [InlineData("http://localhost:5000", "localhost", null)]
```

Comment:
You could generate data for this via `[MemmerData]` function. It would save you some repetition if you're goal is to generate all possible combination from a limited set of inputs.

Reply:
Done